Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -110,7 +110,21 @@ private extension ReceiptRenderer {
     private func summaryTable() -> String {
         var summaryContent = "<table>"
         for line in content.lineItems {
-            let stripedTitle = line.title.htmlStripped()
+            var variations = ""
+            if !line.attributes.isEmpty {
```

Comment:
Not a blocker. In general, `line.attributes.isNotEmpty` is preferred, as it is slightly more readable.

Reply:
Please correct me if I'm wrong. `isNotEmpty` is an extension placed in the `WooCommerce` module. `Hardware` probably don't and should not depend on that module. So it's either we duplicate this function in the `Hardware` module or do a bigger change e.g. moving those extensions to another module that will be available for all modules. wdyt?

Reply:
Argh, good point... Then ignore my comment please. I do not see that much value in either moving this code down to Hardware, or in adding an extension just for this.